docs/platforms/react-native/metrics/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Set Up Metrics
+description: "Learn how to measure the data points you care about by configuring Metrics in your React Native app."
+---
+
+<PlatformContent includePath="metrics/version-support-note" />
+
+Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
+
+## Configure Metrics
+
+<PlatformContent includePath="metrics/configure" />
+
+## Emit a Counter
+
+Counters are one of the more basic types of metrics and can be used to count certain event occurrences.
+
+To emit a counter, do the following:
+
+<PlatformContent includePath="metrics/counter-snippet" />
+
+## Emit a Distribution
+
+Distributions help you get the most insights from your data by allowing you to obtain aggregations such as `p90`, `min`, `max`, and `avg`.
+
+To emit a distribution, do the following:
+
+<PlatformContent includePath="metrics/distribution-snippet" />
+
+## Emit a Set
+
+Sets are useful for looking at unique occurrences and counting the unique elements you added.
+
+To emit a set, do the following:
+
+<PlatformContent includePath="metrics/set-snippet" />
+
+## Emit a Gauge
+
+Gauges let you obtain aggregates like `min`, `max`, `avg`, `sum`, and `count`. They can be represented in a more space-efficient way than distributions, but they can't be used to get percentiles. If percentiles aren't important to you, we recommend using gauges.
+
+To emit a gauge, do the following:
+
+<PlatformContent includePath="metrics/gauge-snippet" />
+
+## Supported Performance Metric Units
+
+You can pass a unit as a optional parameter of the third argument of the `increment`, `distribution`, `set`, and `gauge` methods.
+
+Units augment metric values by giving meaning to what otherwise might be abstract numbers. Adding units also allows Sentry to offer controls - unit conversions, filters, and so on - based on those units. For values that are unitless, you can supply an empty string or `none`.
+
+These following units are understood by the Sentry backend, but you can supply any arbitrary units you want.
```

Comment:
```suggestion
Adding a unit as a metric parameter will give meaning to what may otherwise look like abstract numbers. It also allows Sentry to offer controls - unit conversions, filters, and so on - based on the unit you select. You can pass a unit as an optional parameter of the third argument of the `increment`, `distribution`, `set`, and `gauge` methods. If a value has no units, you can supply an empty string or `none`.

The following units are understood by the Sentry backend, but you can add any arbitrary unit you want.
```